### PR TITLE
Updates to side pane UI

### PR DESCRIPTION
**Pull request by ciduarte**
Added a cancel button to 'add/set editor',
made the select field a paragon component,
CSS edits made to clean up trickle down of
PageContainer on other pages.

DISCO-1050

### CODE REVIEW

**Comment by ciduarte** [comment]
Screen Shot 2019-08-05 at 10 01 50 AM
Screen Shot 2019-08-05 at 10 02 04 AM

File: src/components/SidePanes/UsersPane.jsx [hunk 1]
```diff
@@ -47,6 +49,13 @@ class UsersPane extends React.Component {
     });
   }
 
+  cancelAddUser() {
+    this.setState({
+      addingUser: false,
+      newEditorChoice: 0,
```
Comment (Dillon-Dumesnil) [inline src/components/SidePanes/UsersPane.jsx]: What does setting this to `0` mean? Like, I know it is supposed to be a user id, but is there a chance we are going to send this 0 anywhere? If so, what is the impact of that? Is it better to send `undefined` or `null`?

File: src/components/SidePanes/UsersPane.test.jsx [hunk 2]
```diff
@@ -112,9 +112,6 @@ describe('UsersPane', () => {
     const startAddButton = wrapper.find('.usersPane-startAdd');
     startAddButton.simulate('click');
 
-    const select = wrapper.find('select');
```
Comment (Dillon-Dumesnil) [inline src/components/SidePanes/UsersPane.test.jsx]: Is this not possible to keep anymore? I could see this check still having value.

File: src/components/CreateCourseRunPage/index.jsx [hunk 3]
```diff
@@ -129,8 +129,9 @@ class CreateCourseRunPage extends React.Component {
     const showSpinner = !startedFetching || courseInfo.isFetching || courseRunOptions.isFetching;
     const showForm = startedFetching && !courseInfo.isFetching && !courseRunOptions.isFetching &&
       !courseInReview;
-    const sortedRunLabels = course_runs && this.parseCourseRunLabels(course_runs);
-    const defaultRun = sortedRunLabels ? sortedRunLabels[0].value : '';
+    const sortedRunLabels = (showForm &&
```
Comment (ciduarte) [inline src/components/CreateCourseRunPage/index.jsx]: Found a bug where the select field on the 'add course run' page will cause an error for courses without existing runs. Wanted to push a fix for this edge case out with these edits prior to bug bash.

File: src/components/SidePanes/UsersPane.test.jsx (hunk 2 above)
Comment (ciduarte) [reply src/components/SidePanes/UsersPane.test.jsx]: I was actually able to select the option rather than the entire select field.

If you're curious, follow up to your suggestion of debugging the wrapper: I was running into issues using `wrapper.find()` because the InputSelect component from Paragon is actually labeled as `<asInput(withDeprecatedProps(InputSelect)) />`, so neither `wrapper.find('select')` or `wrapper.find('InputSelect')` was doing the trick. Selecting the option directly is more straight forward, and will (hopefully) help if/when we switch to Paragon V2.
